src/LogzioShipper/file_handler.py

```diff
@@ -41,7 +43,12 @@ def __init__(self, file_name: str, file_stream: IOBase, file_size: int) -> None:
         self._file_size = file_size
         self._filter_date = self._get_filter_date()
         self._filter_date_json_path = self._get_filter_date_json_path()
-        self._file_parser = self._get_file_parser()
+
+        try:
+            self._file_parser = self._get_file_parser()
```

Comment:
Why if we failed to recognize what is it we dont send it as a text ?

Reply:
fix it to use text format when csv delimiter cant be found